canonical/jaas-dashboard
# WD-2379 - Include custom icons in CSS

## Done

- Include custom icons in CSS.

## QA

- Pull code
- Run `dotrun clean && dotrun serve`
- Open http://0.0.0.0:8036/
- Deploy a unit with a subordinate.
- Go to the unit list for the application.
- Check that the 'Run action', 'View action logs' and the L icon next to the subordinate all appear. 

## Details

Fixes: #944.
https://warthogs.atlassian.net/browse/WD-2379

## Screenshots

<img width="324" alt="Screenshot 2023-03-02 at 3 59 59 pm" src="https://user-images.githubusercontent.com/361637/222335452-701dcd8d-428a-404e-a09f-1250e35975bd.png">
